packages/core/components/QuerySidebar/Query.tsx

```diff
@@ -128,9 +130,33 @@ export default function Query(props: QueryProps) {
                 )}
                 {!!queryComponents.filters.length && (
                     <p className={styles.displayRow}>
-                        <strong>Filter:</strong>{" "}
-                        {queryComponents.filters
-                            .map((filter) => `${filter.name}: ${filter.value}`)
+                        <strong>Filter{queryComponents.filters.length > 1 ? "s" : ""}:</strong>{" "}
+                        {/* Show a condensed version of the filters list with counts instead of values */}
+                        {Object.entries(
+                            queryComponents.filters.reduce((accum, filter) => {
```

Comment:
This is pretty busy to put into the component tree here - could we pull this out as a function that this calls instead?